composer/datasets/in_context_learning_evaluation.py

```diff
@@ -531,7 +540,7 @@ def prep_examples(self, num_fewshot: int, prompt_string: str, example_delimiter:
                         'choices'], self.samples[fewshot_idx]['gold']
                     if len(preamble) > 0:
                         query = f'{example_delimiter}{query}'
-                    preamble += f'{query}{continuation_delimiter}{choices[gold_idx]}'
+                    preamble += f'{query}{continuation_delimiter}{choices[gold_idx]}'  # pyright: ignore [reportGeneralTypeIssues]
```

Comment:
why does this need a new type ignore? i don't think it should, so might indicate a bug?

Reply:
I think because of the strip_data call `gold_idx` now has an unknown type?

Reply:
ok, can you assert that its an int instead of type ignore?